### ISSUE TITLE
Accessibility Testing for Benefits Management Team 2 - Claim Status Tool - Document Status

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#127730]
### Product information

- [x] Team name, product name, and feature name have been added to the title of this issue.
- [x] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@bacitracin

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")
- [x] (Mobile app only) Color contrast ratios are met for all supported dark, contrast, and inverted color modes
- [x] (Mobile app only) All elements retain their meaning for all supported dark, contrast, and inverted color modes

### How did color testing go?

No color & contrast issues identified

### Who completed the axe scans?

@bacitracin

### axe checks

- [x] Each page or screen has been scanned using axe (results shared below)
- [x] axe is integrated into your end-to-end testing

## Happy Path / User 15 screens
https://staging.va.gov/track-claims/your-claims/600760833/files
<img width="1000" height="1352" alt="Image" src="https://github.com/user-attachments/assets/780900b4-73a4-4557-8f29-b77a8afc644a" />

https://staging.va.gov/track-claims/your-claims/600759243/files
<img width="1000" height="1166" alt="Image" src="https://github.com/user-attachments/assets/cc67f6eb-82c1-4dbd-a44c-17ef79f22659" />

https://staging.va.gov/track-claims/your-claims/600820579/files
<img width="1000" height="1275" alt="Image" src="https://github.com/user-attachments/assets/05de19e3-438c-4ccf-b151-9aa3f70e889f" />

https://staging.va.gov/track-claims/your-claims/files-we-couldnt-receive
<img width="1000" height="1177" alt="Image" src="https://github.com/user-attachments/assets/c5d58f65-b18a-457c-ad46-8928d7cf458e" />

## Type 1 Error / User 276 Screens
https://staging.va.gov/track-claims/your-claims/600679477/files
Successful doc upload with alert
<img width="1000" height="1012" alt="Image" src="https://github.com/user-attachments/assets/1f4cd5c7-cc5b-4ffc-a4e1-eb07a6054713" />

https://staging.va.gov/track-claims/your-claims/600679477/files
After successful doc upload try to upload the same file again
<img width="1000" height="1026" alt="Image" src="https://github.com/user-attachments/assets/ed459df8-0ac0-4277-8423-70b51034a3be" />

## Type 2 Error / User 44 Screens
https://staging.va.gov/track-claims/your-claims
<img width="1000" height="877" alt="Image" src="https://github.com/user-attachments/assets/81c460d6-1373-42bc-a55b-f850f1e3fafd" />

https://staging.va.gov/track-claims/your-claims/600810943/status
<img width="1000" height="820" alt="Image" src="https://github.com/user-attachments/assets/1a3e8f8e-339d-4569-9122-d13782333661" />

https://staging.va.gov/track-claims/your-claims/600810943/files
<img width="1000" height="809" alt="Image" src="https://github.com/user-attachments/assets/03d84de6-ca96-4e1d-9459-309ccb8fbc6e" />

https://staging.va.gov/track-claims/your-claims/600636741/status
<img width="1000" height="867" alt="Image" src="https://github.com/user-attachments/assets/69417a8f-3d81-4b9f-99e8-40fe419286aa" />

### How did axe testing go?

No issues identified with Axe DevTools scans

### Who completed the content zoom and reflow test?

@bacitracin

### Content zoom and reflow checks

- [x] (Web only) All page elements are readable and usable at 200% zoom
- [x] (Web only) All page elements are readable and usable at 300% zoom
- [x] (Web only) All page elements are readable and usable at 400% zoom
- [ ] (Mobile app only) All page elements are readable and usable when device text is at the maximum possible size

### How did content zoom and reflow testing go?

An earlier test found that at 300+% zoom the card titles weren't wrapping to the next line. That [issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/127280) has been fixed and merged in. 

### Who completed the keyboard navigation test?

@bacitracin

### Keyboard navigation checks

- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [x] All elements under focus have a visible focus indicator
- [x] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

Found 2 quirks with the success & error alerts that are documented in [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/127648). The team is currently working on the ticket. 
1. Focus order - for the green success alert. Keyboard focus goes down to the link below and then back up to the top of the alert for the X close button. Navigating via screen reader does this as well, probably the X is last in the HTML order.
2. Receiving focus - for both alerts. When the alerts appear, focus moves to them, and the user can navigate via keyboard to the link & X close button, but if they navigate out of the alert, they cannot return to the link/buttons inside the alert again. 

### Do you have any other results to share?

I don't know if this is part of the work from this feature or not, but when testing with VoiceOver I found two areas in the CST where there are just empty paragraphs. Two issues have been filed.
- One on the request for evidence page [issue ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/127714)
- One under the CST's Recent Activity section, when there are the blue alerts. [issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/127715)
